### DISCUSSION TITLE
Tooltip Events Not Firing

### DISCUSSION THREAD

**Discussion by sblawrie** [Akryum/v-tooltip#342] [Q&A] [answered]
I've tried every which way to listen to the events that v-tooltip is supposed to emit, and haven't been able to see / catch any. There are no examples in the documentation. Is my code below incorrect? The `alert()` is not fired. Is there an example somewhere? Thank you!

```
<h3 @show="function(e) {alert('success!');}" v-tooltip="{
    html: true,
    content: 'hi',
    offset: 50,
    placement: 'top',
    autoHide: false,
    classes: ['tooltip-offset'],
}">Test Hover</h3>
```

**Comment by Akryum** [comment, marked as answer]
Directives don't emit events, maybe you should use the component instead.
